src/.github/workflows/build.yml

```diff
@@ -25,22 +25,22 @@ jobs:
           docker build .
 
   build-go:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Check out repository
         uses: actions/checkout@v3
 
       - name: Setup Go environment
-        uses: actions/setup-go@v3.0.0
+        uses: actions/setup-go@v3.3.1
         with:
-          go-version: "1.17.6"
+          go-version: "1.9"
```

Comment:
Why do you downgrade the [version of go](https://go.dev/blog/go1.19)?

Reply:
I meant "19" sorry